### DISCUSSION TITLE
Big thanks!

### DISCUSSION THREAD

**Discussion by jlodew0** [xblax/flashforge_ad5m_klipper_mod#11] [Show and tell]
Since people forget this a lot of times, I took the opportunity to be the first.

To @xblax and @g992 ; I want to say a big big big thanks!
I consider both of you to be the first brick layers of briging vanilla klipper to the 5m series and share this with a broader (opensource) community!

I also love the way you guy's interact with everyone asking questions and trying to help.
It's realy nice to see how projects like this forms nice and warm communities of people with interests alike!

So for that, many thanks to the both of you! ❤️

**Comment by g992** [comment]
Thank u bro 🤚

**Reply by jlodew0** [reply]
no, thank you! Realy 🥇 
p.s. I have broken my mb as well. I realy do HATE it when they use hot glue or whatever glue to 'support' connectors.
I ripped of the ufl connector for the wifi of the motherboard, together with the traces for it. (sigh) 
Not that I used the wifi, but still.

**Comment by AJolly** [comment]
Yes agreed, I wouldn't have bought my 5M if it wasn't for this project!

**Comment by xblax** [comment]
Thank you as well everyone for helping with this project! I had already looked at this printer for a while but wouldn't have bought the printer if @g992 didn't start documenting the hardware on Github.

**Comment by DougJoseph** [comment]
**Many thanks from me as well. I appreciate you all so much for doing this mod!!**

**Comment by MASalmeron** [comment]
Thanks to the whole team Flashforge_ad5m_klipper_mod, I left you your sample on my 5M PRO, bravo! I created a video thanks to your work for Spanish-speaking makers, thanks!!
![photo_2024-09-11_22-00-00](https://github.com/user-attachments/assets/d6d9b983-41e6-4db9-a853-746632881602)
